test/presets/cloudflare-pages.test.ts

```diff
@@ -12,7 +12,7 @@ describe("nitro:preset:cloudflare-pages", async () => {
   testNitro(ctx, () => {
     const mf = new Miniflare({
       modules: true,
-      scriptPath: resolve(ctx.outDir, "_worker.js"),
+      scriptPath: resolve(ctx.outDir, "_worker.js", "index.js"),
```

Comment:
Do we always need a path like this as `.../_worker.js/index.js`? (double extension seems strange but fine if it is only allowed convention)

Reply:
yeah it's the convention unfortunately :sweat: 

later this might be configurable but for now it has to be like this :sweat: